src/webgl/p5.Geometry.js

```diff
@@ -147,6 +156,69 @@ p5.Geometry.prototype.averagePoleNormals = function() {
 };
 
 /**
+ * Create a 2D array for establishing stroke connections
+ * @return {p5.Geometry}
+ */
+p5.Geometry.prototype._makeTriangleEdges = function() {
+  if (Array.isArray(this.strokeIndices)) {
+    for (var i=0, max=this.strokeIndices.length; i<max; i++) {
+      this.edges.push(this.strokeIndices[i]);
+    }
+  }
+  else {
+    for (var j=0, amt=this.faces.length; j<amt; j++) {
```

Comment:
this for loop instantiation is hard to read, can we just write:
`for (var j=0; j<this.faces.length; j++) {`

Reply:
I changed this for-loop as per your request but just wanted to clarify why I was doing that. These loops are sometimes going over arrays that are many 10,000's of items long, and if a variable isn't established with the length of the array then the length of the array is checked 10,000's of times. It is a minor performance boost in most of the manual tests though and I agree that clarity and ease of entry for new contributors is more important than performance here!